src/main/java/net/coderbot/iris/uniforms/CameraUniforms.java

```diff
@@ -28,7 +28,7 @@ public static void addCameraUniforms(UniformHolder uniforms, FrameUpdateNotifier
 	}
 
 	private static int getRenderDistanceInBlocks() {
-		return client.gameRenderer.getViewDistance();
+		return client.gameRenderer.getViewDistance() * 16;
```

Comment:
`GameRenderer#<init>`:
![image](https://user-images.githubusercontent.com/18603248/127719460-3fafc203-3c75-4bc0-8c96-86198d5a9963.png)
```diff
- return client.gameRenderer.getViewDistance() * 16;
+ return client.gameRenderer.getViewDistance();
```

Reply:
Where is this you say?

Reply:
`GameRenderer#renderWorld`